flink-table/flink-sql-gateway/pom.xml

```diff
@@ -47,6 +47,7 @@
             <groupId>org.apache.flink</groupId>
             <artifactId>flink-sql-gateway-api</artifactId>
             <version>${project.version}</version>
+			<optional>${flink.markBundledAsOptional}</optional>
```

Comment:
nit: 
seems alignment should be fixed

Reply:
that's an interesting one: `flink-table/flink-sql-gateway/pom.xml` has a mixture of spaces and tabs. Other pom.xml files use mainly tabs as whitespaces. I would have expected that this is also covered by the spotless plugin or checkstyle. I have to do more research here.